### PR TITLE
remove unused stuff

### PR DESCRIPTION
**Pull request by UFOMelkor**
Mainly unused imports

### CODE REVIEW

File: src/Hal/Component/OOP/Resolver/NameResolver.php [hunk 1]
```diff
@@ -52,7 +52,6 @@ public function resolve($name, $currentNamespace = '\\')
         }
 
         return rtrim($currentNamespace, '\\').'\\'.$name;
-        return $currentNamespace.$name;
```
Comment (Halleck45) [inline src/Hal/Component/OOP/Resolver/NameResolver.php]: why did you remove this code ?

Comment (UFOMelkor) [inline src/Hal/Component/OOP/Resolver/NameResolver.php]: Because of the return in the line above.

Comment (Halleck45) [inline src/Hal/Component/OOP/Resolver/NameResolver.php]: Aaah yes, of course :)

**Comment by Halleck45** [comment]
Thank's !
